crates/bevy_ecs/src/world/mod.rs

```diff
@@ -917,6 +918,67 @@ impl World {
             column.check_change_ticks(change_tick);
         }
     }
+
+    /// Runs a system in a sequential, blocking fashion on the `World`
```

Comment:
why "in a sequential" when there is only one?

Reply:
I want to be very explicit about the fact that this is going to be slower than standard system runs. I think it's probably clearer without it though.